src/main/java/teammates/ui/controller/CreateImageUploadUrlAction.java

```diff
@@ -0,0 +1,40 @@
+package teammates.ui.controller;
+
+import com.google.appengine.api.blobstore.BlobstoreFailureException;
+
+import teammates.common.exception.EntityDoesNotExistException;
+import teammates.common.util.Const;
+import teammates.common.util.GoogleCloudStorageHelper;
+
+public class CreateImageUploadUrlAction extends Action {
+
+    @Override
+    protected ActionResult execute() throws EntityDoesNotExistException {
+        return createAjaxResult(createImageUploadUrlPageData());
+    }
+
+    protected final CreateImageUploadUrlAjaxPageData createImageUploadUrlPageData() {
```

Comment:
This is actually quite trivial, but the method name is almost identical with the return type and it can cause confusion. Either make it `createCreate...` (equally confusing, though) or `getCreate...`.